prowler/lib/check/check.py

```diff
@@ -566,6 +557,17 @@ def execute(
     # Report the check's findings
     report(check_findings, audit_output_options, audit_info)
 
+    if os.environ.get("PROWLER_REPORT_LIB_PATH"):
+        try:
+            logger.info("Using custom report interface ...")
+            lib = os.environ["PROWLER_REPORT_LIB_PATH"]
+            outputs_module = importlib.import_module(lib)
+            curstom_report_interface = getattr(outputs_module, "report")
```

Comment:
I think you have to handle more than a generic `Exception` here not to abort the execution if the `"PROWLER_REPORT_LIB_PATH"` is not present or if the `import` or `getattr` fails.

Reply:
Since the  general `report` function is imported always we actually want Prowler to exit the execution if something wrong happens there